released/packages/coq-lens/coq-lens.1.0.1+8.11/opam

```diff
@@ -0,0 +1,29 @@
+opam-version: "2.0"
+synopsis: "Generation of lenses for record datatypes"
+maintainer: "gregory@bedrocksystems.com"
+authors: [
+  "Gregory Malecha <gregory@bedrocksystems.com>"
+]
+homepage: "https://github.com/bedrocksystems/coq-lens"
+dev-repo: "git://github.com/bedrocksystems/coq-lens.git"
+bug-reports: "https://github.com/bedrocksystems/coq-lens/issues"
+license: "LGPL2.1+BedRock"
+build: [
+  [make "-j%{jobs}%"]
+]
+install: [
+  [make "install"]
+]
+depends: [
+  "ocaml"
+  "coq" {>= "8.11" & < "8.12~"}
+  "coq-metacoq-template" { = "1.0~beta1+8.11" }
+]
+tags: [
+  "logpath: Lens"
+  "date: 2020-03-29"
```

Comment:
I've copied these tags from the current package, but I'm not sure of the semantics.